source/_docs/scripts.markdown

```diff
@@ -886,6 +886,10 @@ response data must contains a mapping of key/value pairs.
   response_variable: "my_response_variable"
 ```
 
+{% warning %} 
+The response_variable will not be returned when the script mode is set to queued. 
```

Comment:
Have you confirmed this is intended functionality and thus should be documented like this? As in, to me, this sounds like a bug, which should be reported, looked at and fixed.

Reply:
Hey Frenk :)

I wanted this updated to reflect the reality of how response variables work under different modes. My assumption was that it was by design because the issue raised in November 2023, with 12 thumbs-ups, was not responded to for 9 months
https://github.com/home-assistant/core/issues/104218

The issue was confirmed as still present up to August 2024, after which triage closed the issue as stale

@toxikman requested you reopen the issue on August 18th

If you do believe this is as bug as several of us do, perhaps you might be able to reopen #104218

If not, I feel it would be beneficial to users to know how the software functions.

Thanks in advance

Teska

Reply:
Re-opened.